### DISCUSSION TITLE
Adding Users from Outside the Org to Projects

### DISCUSSION THREAD

**Discussion by austinbenerdy** [community/community#9258] [Projects and Issues]
Creating a Project board within my organization, we would like to give access to specific people from outside of our organization. Our Project is private. 

I am wondering if there's any way, or any plans to make it possible in the future, to allow invited users from outside of an organization to access an organization's private project.

**Comment by hawksight** [comment]
For classic projects I found [this link](https://docs.github.com/en/organizations/managing-access-to-your-organizations-project-boards/adding-an-outside-collaborator-to-a-project-board-in-your-organization).
But for the newer style projects I couldn't find any documentation on it.
